OpenSim/Examples/DataTable/example.cpp

```diff
@@ -0,0 +1,61 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  example.cpp                           *
```

Comment:
Currently DataTable has its own directory under examples. More files can be added as examples within this directory by naming them example1.cpp, example2.cpp and so on.
I can change this to have just one examples file if desired and name it what you suggest.